orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/general/EqualTo.feature

```diff
@@ -5,351 +5,41 @@ Feature: User can specify that a value is equalTo a required value
 
```

Comment:
You mentioned that these are duplicated tests that you've deleted. Where are the tests that they duplicate?

Reply:
duplicated within this feature.